tutorials/notebooks/README.md

```diff
@@ -0,0 +1,140 @@
+# Tutorials
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Keras Tutorials](#keras-tutorials)
+  - [Post-Training Quantization](#keras-ptq)
+  - [Gradient-Based Post-Training Quantization](#keras-gptq)
+  - [Quantization-Aware Training](#keras-qat)
+  - [Structured Pruning](#keras-pruning)
+  - [Export Quantized Models](#keras-export)
+  - [Debug Tools](#keras-debug)
+- [Pytorch Tutorials](#pytorch-tutorials)
+  - [Quick-Start with Torchvision](#pytorch-quickstart-torchvision)
+  - [Post-Training Quantization](#pytorch-ptq)
+  - [Quantization-Aware Training](#pytorch-qat)
+  - [Data Generation](#pytorch-data-generation)
+  - [Export Quantized Models](#pytorch-export)
+
+## Introduction
+Dive into MCT with our collection of tutorials, covering a wide 
+range of compression techniques for Keras and Pytorch models. We provide
+both Python scripts and interactive Jupyter notebooks for an
+engaging and hands-on experience.
+
+
+## Keras Tutorials
+
+<details id="keras-ptq">
+  <summary>Post-Training Quantization</summary>
```

Comment:
Post-Training Quantization (PTQ). Do this for QAT and all acronyms you use in the features section.